.opencode/tool/ollama-queue.ts

```
@@ -0,0 +1,1116 @@
+
+  try {
+    let result: unknown;
+    let cacheKey: string | null = null;
+    const startTime = now();
+
+    switch (job.type) {
+      case 'generate':
+        if (!job.prompt) throw new Error('Generate job missing prompt');
+
+        // Check cache first
+        const cachedResult = await checkCache(job.prompt, job.modelName, job.type);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)